### PR TITLE
Style desktop verson

### PR DESCRIPTION
**Pull request by dd-obua**
In this project I wrote CSS styles in a media query to implement page appearance on desktops and devices of screen sizes 768px and larger.

### CODE REVIEW

File: css/desktop.css [hunk 1]
```diff
@@ -0,0 +1,179 @@
+@media screen and (min-width: 768px) {
+  .headline,
+  .about-me {
+    background-image: none;
+  }
+
+  .about-me__heading::before,
```
Comment (iambenkis) [inline css/desktop.css:7]: Keep doing the great job 👍

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @dd-obua , @BHObserver 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️  Desktop version
✔️ Use of Flex and grid  \

## Require changes ♻️ 

- [ ] Kindly fix the broken link attached to your `live demo` text . if you the live demo is not available I will suggest to put a `N/A` text instead. It will make your README file more professional 😄 
Capture1

Capture


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: css/desktop.css (hunk 1 above)
Comment (dd-obua) [reply css/desktop.css:7]: Thank you so much for your feedback. However, I do not understand this part of the feedback. Kindly give some detail.

Comment (iambenkis) [reply css/desktop.css:7]: It's a kind of encouragement 😄

Comment (dd-obua) [reply css/desktop.css:7]: Oh thank you. I have corrected the README file. I'm just going to ask for the next review. Cheers.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @dd-obua, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Good design ✔️ 

 ## Required Changes ♻️
- [ ] I cannot leave comment there so I am leaving it here. As requested by the previous reviewer kindly fix your live-demo text please put N/A text instead. It will make your README file more professional 😄. You made changes above. So please provide N/A in actual live-demo that is below `key feature`
- [ ] [Optional] If the requirement say not to deploy the app then you can remove the live-demo as it make more professional README file.
### Your README 
his

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -72,7 +72,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- Live Demo Link (N/A) coming soon.
+- N/A
```
Comment (TadesseDev) [inline README.md]: - [Optional] Dear @dd-obua, good job making the changes as requested by the previous reviewers. But for the live demo section, It's required to have a working link or no live demo section. Please consider removing the live demo section unless your project has a live demo.

**Review by TadesseDev — APPROVED** [review]
### Hi @dd-obua 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
240590909-9d125ce4-1a1e-4269-91d9-a07846cecea3




## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**